app/index.html

```diff
@@ -35,81 +35,79 @@
     <body>
         <header>
             <nav class="container navbar navbar-expand-lg navbar-dark">
-                <a class="navbar-brand" href="#">chemiscope</a>
-                <button
-                    class="navbar-toggler"
-                    type="button"
-                    data-toggle="collapse"
-                    data-target="#navbar"
-                    aria-controls="navbar"
-                    aria-expanded="false"
-                    aria-label="Toggle navigation"
-                >
-                    <span class="navbar-toggler-icon"></span>
-                </button>
-
-                <div class="collapse navbar-collapse" id="navbar">
-                    <ul class="navbar-nav mr-auto">
-                        <li class="nav-item">
-                            <a class="nav-link" href="https://chemiscope.org/docs/"><i class="fa fa-book"></i> Documentation</a>
-                        </li>
-
-                        <li class="nav-item">
-                            <a class="nav-link" data-toggle="modal" href="#about"><i class="fa fa-user"></i> About</a>
-                        </li>
-
-                        <li class="nav-item hide-if-standalone">
-                            <a class="nav-link" data-toggle="modal" href="#sources"><i class="fa fa-download"></i> Sources</a>
-                        </li>
-
-                        <li class="nav-item dropdown hide-if-standalone">
-                            <a
-                                class="nav-link"
-                                href="#"
-                                id="examples"
-                                role="button"
-                                data-toggle="dropdown"
-                                aria-haspopup="true"
-                                aria-expanded="false"
-                            >
-                                <i class="fa fa-images"></i> Examples
-                            </a>
-                            <div class="dropdown-menu" aria-labelledby="examples">
-                                <span class="dropdown-item clickable" onclick="CHEMISCOPE_APP.loadExample('Arginine-Dipeptide');">
-                                    Arginine-Dipeptide
-                                </span>
-                                <span class="dropdown-item clickable" onclick="CHEMISCOPE_APP.loadExample('CSD-1000R');"> Chemical Shieldings </span>
-                                <span class="dropdown-item clickable" onclick="CHEMISCOPE_APP.loadExample('Qm7b');"> Qm7b </span>
-                                <span class="dropdown-item clickable" onclick="CHEMISCOPE_APP.loadExample('Azaphenacenes');"> Azaphenacenes </span>
-                                <span class="dropdown-item clickable" onclick="CHEMISCOPE_APP.loadExample('Zeolites');"> Zeolites </span>
-                            </div>
-                        </li>
+                <div class="container-fluid">
```

Comment:
should we make the div above a `container-fluid` instead of adding another one?

Reply:
I don't think so, that's a breaking change described in [the migration guide](https://getbootstrap.com/docs/5.0/migration/#navbars) and [docs](https://getbootstrap.com/docs/5.0/components/navbar/). I think I should move the `container` class to the `header` element though.